daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

```diff
@@ -27,6 +27,8 @@ spec:
     value: "dapr-kafka.myapp.svc.cluster.local:9092"
   - name: consumerGroup # Optional. Used for input bindings.
     value: "{namespace}"
+  - name: consumerID # Optional. If not supplied, runtime will create one.
+    value: "topic1"
```

Comment:
I would not call this "topic1", since that is confusing. Instead use "channel1" as an example